HW_API/index.js

```diff
@@ -0,0 +1,43 @@
+
+const imageGalleryContainer = document.querySelector('.images-gallery');
+
+class ImagesGallery {
+    constructor() {
+        this.images = [];
```

Comment:
```suggestion
        this.images = [];
        this.imageGalleryContainer = imageGalleryContainer;
```
також, в ідеалі, треба перевірити чи те що ми сюди передали дом елемент, чи ні, подумайте, якщо буде час, як це реалізувати